modules/xml/src/main/java/org/mule/module/xml/transformer/XsltTransformer.java

```diff
@@ -141,6 +150,23 @@ protected void doInitialise() throws InitialisationException
         {
             throw new InitialisationException(te, this);
         }
+
+        if (logger.isWarnEnabled() && !isTransformerPoolEvictionEnabled())
+        {
+            logger.warn("Transformer pool eviction is disabled.");
```

Comment:
My thinking was that since transformer pools have no eviction, by deafult, users should be warned about this. However, I see now that this warning would be outputted independent of whether eviction was purposely disabled by a user or just as part of the default configuration.